htdocs/js/jquery.quickreply.js

```diff
@@ -208,6 +208,7 @@ jQuery(document).ready(function($) {
         // take a random number, ignoring the "(default)" option
         var randomnumber = Math.floor(Math.random() * (iconslist.length-1) ) + 1;
         iconslist.selectedIndex = randomnumber;
+        $("#prop_picture_keyword").trigger("change");
```

Comment:
on second thought, let's not re-select.

```suggestion
        $( iconslist ).trigger("change");
```